examples/story_diffusion/README.md

```diff
@@ -0,0 +1,63 @@
+# StoryDiffusion: Consistent Self-Attention for Long-Range Image and Video Generation
+This is an efficient MindSpore implementation of [StoryDiffusion](https://github.com/HVision-NKU/StoryDiffusion/tree/main)
+
+## Introduction
+
+StoryDiffusion can generate consistent images and videos given a set of text prompts. The major contributions are two-fold:
+
+1. Consistent self-attention, a training-free and pluggable module with all SD1.5 and SDXL-based image diffusion models. Given at least 3 text prompts, it can generate images with consistent subjects and good text-vision alignment.
+2. Motion predictor for long-range video generation, which can predict motion between condition images.
+
+The current implementation only supports text-to-image generation with consistent self-attention.
+
+## TODOs
+- [x] Comic Generation Inference script
+- [ ] Gradio demo of comic generation
```

Comment:
are you going to deploy the demo?

Reply:
The demo is ready now. I have made some edits to `mindone/diffusers/pipelines/stable_diffusion_xl/pipeline_stable_diffusion_xl.py` to solve some errors.